apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-spring5/src/test/java/co/elastic/apm/agent/springwebflux/AbstractServerInstrumentationTest.java

```diff
@@ -133,7 +133,7 @@ private void hello(boolean expectHeaders) {
 
             assertThat(headersCount)
                 .describedAs("unexpected headers count")
-                .isEqualTo(6);
+                .isGreaterThanOrEqualTo(6);
```

Comment:
[for reviewer] there is now 7 headers when only 6 were previously reported, here we don't really have a strict assertion on all of them, only a few are expected with specific values as seen in the assertions just below.